incubator/hnc/test/e2e/quickstart_test.go

```diff
@@ -249,4 +249,39 @@ spec:
 		RunShouldContain(expected, defTimeout, "kubectl hns tree", nsTeamA)
 		RunShouldContain("ActivitiesHalted (ParentMissing):", defTimeout, "kubectl hns describe", nsStaging)
 	})
+
+	It("Should demonstrate exceptions", func() {
+		// set up initial structure
+		CreateNamespace(nsOrg)
+		CreateSubnamespace(nsTeamA, nsOrg)
+		CreateSubnamespace(nsTeamB, nsOrg)
+
+		MustRun("kubectl -n", nsOrg, "create secret generic my-secret --from-literal=password=iamacme")
+		// allow secret to propagate
+		MustRun("kubectl hns config set-resource secrets --mode Propagate --force")
+		// wait 2 seconds to give time for secret to propogate
+		time.Sleep(2 * time.Second)
+		// check that the secrete has been propagated to both subnamespaces
+		RunShouldContain("my-secret", defTimeout, "kubectl -n", nsTeamA, "get secrets")
+		RunShouldContain("my-secret", defTimeout, "kubectl -n", nsTeamB, "get secrets")
+
+		// add exceptions annotation
+		MustRun("kubectl annotate secret my-secret -n", nsOrg, "propagate.hnc.x-k8s.io/treeSelect=!team-b")
+		// check taht the secret is no longer accessible from team-b
+		RunShouldNotContain("my-secret", defTimeout, "kubectl -n", nsTeamB, "get secrets")
+
+		// delete secret and re-create from the yaml file
+		MustRun("kubectl delete secret my-secret -n", nsOrg)
```

Comment:
Maybe wait for it to be removed from `teamA` first

Reply:
+1, add `RunShouldNotContain` teamA before this line. Or annotate the secret yaml below with `treeSelect: team-b`.

Reply:
added the check for removal